Hey @ajaymahadeven, thanks for this PR. Minor linting failures - can you please address?

```
_________________________ TestCodeQuality.test_flake8 _________________________
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.13.11\x64\Lib\unittest\case.py", line 58, in testPartExecutor
    yield
  File "C:\hostedtoolcache\windows\Python\3.13.11\x64\Lib\unittest\case.py", line 651, in run
    self._callTestMethod(testMethod)
    ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "C:\hostedtoolcache\windows\Python\3.13.11\x64\Lib\unittest\case.py", line 606, in _callTestMethod
    if method() is not None:
       ~~~~~~^^
  File "D:\a\_work\1\s\tests\test_code_quality.py", line 54, in test_flake8
    raise AssertionError(
        f'flake8 validation failed:\n{output}') from None
AssertionError: flake8 validation failed:
.\tests\test_http_asgi.py:73:41: E127 continuation line over-indented for visual indent
```

--- a/tests/test_http_asgi.py
+++ b/tests/test_http_asgi.py
@@ -70,7 +70,7 @@ class MockAsgiApplication:
             assert isinstance(scope['http_version'], str)
 
             assert scope['method'] in ['POST', 'GET', 'PUT', 'DELETE', 'PATCH',
-                                        'QUERY']
+                                       'QUERY']
             assert isinstance(scope['method'], str)
 
             assert scope['scheme'] in ['http', 'https']

